### PR TITLE
add: `bottles-v1-deb`

### PR DESCRIPTION
**Pull request by cat-master21**

### CODE REVIEW

**Comment by cat-master21** [comment]
Wine64 has issues apparently https://github.com/pacstall/pacstall-programs/runs/5400356628?check_suite_focus=true#step:9:46747.

**Comment by cat-master21** [comment]
Still as issues? https://github.com/pacstall/pacstall-programs/runs/5400751029?check_suite_focus=true#step:9:46444

**Comment by Henryws** [comment]
> Still as issues? [pacstall/pacstall-programs/runs/5400751029?check_suite_focus=true#step:9:46444](https://github.com/pacstall/pacstall-programs/runs/5400751029?check_suite_focus=true#step:9:46444)

Yeah the issue is with the wine pacscript:
```diff
1,3c1,3
< name="wine"
< pkgname="wine"
< provides="wine-stable wine"
---
> name="wine64"
> pkgname="wine64"
> provides="wine64-stable wine64"
19c19
<           sudo make install DESTDIR=/usr/src/pacstall/wine
---
>           sudo make install DESTDIR=/usr/src/pacstall/wine64
```

Change the wine pacscript accordingly.

**Comment by wizard-28** [comment]
@Henryws I think you'll be able to do the review better on this one.

**Comment by Henryws** [comment]
Does Bottles *need* the latest wine version? Because adding `wine64` as a pacdep will take insanely long building times.

**Comment by cat-master21** [comment]
This is why: https://github.com/pacstall/pacstall-programs/runs/5442664531?check_suite_focus=true#step:9:41 Debian doesn't have wine-stable unlike Ubuntu.

**Comment by wizard-28** [comment]
@Henryws any updates on this?

**Comment by Henryws** [comment]
> @Henryws any updates on this?

Workin on it.

**Review by Henryws — CHANGES_REQUESTED** [review]
bottles doesn't call the wine64 pacdep

**Comment by cat-master21** [comment]
No, but it calls for `wine-stable` which is only on Ubuntu, not on Debian so Debian installation fails but wine64 as a pacdep should solve the issue.

**Comment by Henryws** [comment]
> No, but it calls for `wine-stable` which is only on Ubuntu, not on Debian so Debian installation fails but wine64 as a pacdep should solve the issue.

Add an if statement checking if the OS is debian, and if so, make the pacdep, and if not, add `wine-stable` to `depends`

**Comment by wizard-28** [comment]
@Henryws any updates on this?

**Comment by Henryws** [comment]
> @Henryws any updates on this?

Still waiting on https://github.com/pacstall/pacstall-programs/pull/802#issuecomment-1065930986

**Comment by cat-master21** [comment]
So I fixed this at Bottles at pull requests https://github.com/bottlesdevs/Bottles/pull/1204/files and https://github.com/bottlesdevs/Bottles/pull/1207/files to switch to GitHub actions and use `wine` instead of `wine-stable`.

**Comment by cat-master21** [comment]
The error is from trying to install Bottles-v1 and wine64.

**Comment by Henryws** [comment]
Remove the wine pacscript from this PR, other than that, looks good.

File: packages/bottles-v1-deb/bottles-v1-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name="bottles-v1-deb"
+gives="bottles-v1"
```
Comment (Henryws) [inline packages/bottles-v1-deb/bottles-v1-deb.pacscript:2]: `gives` does not utilize the actual deb name, which is `com.github.mirkobrombin.bottles`

File: packages/bottles-v1-deb/bottles-v1-deb.pacscript [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name="bottles-v1-deb"
-gives="bottles-v1"
-breaks="${gives} ${gives}-git ${gives}-bin ${gives}-app"
+gives="bottles-v1 com.github.mirkobrombin.bottles bottles"
```
Comment (Henryws) [inline packages/bottles-v1-deb/bottles-v1-deb.pacscript:2]: `gives` can only have one key in it, so remove `bottles-v1`

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by cat-master21** [comment]
Name of the deb is different then bottles-v1-deb which Pacstall can't take.

**Pull request closed without merging**